TaskBroker/VstsServerTaskBroker.ResultUploader/VstsServerTaskBroker.ResultUploader.csproj

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <ProjectGuid>{FA90DF4F-36DB-44CC-895D-2B174FE62DAC}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>VstsServerTaskBroker.ResultUploader</RootNamespace>
```

Comment:
Suresh added this as part of uploading the results. Need to come up with a way to consume the external package plus with these add on functionality internally.